### PR TITLE
Removed console error

### PR DESCRIPTION
**Pull request by OliaHrytsak**
Removed catch(console.error) in places, where it was used.

### CODE REVIEW

File: src/containers/edit-profile/profile-tab/ProfileTab.tsx [hunk 1]
```diff
@@ -65,16 +69,23 @@ const ProfileTab: FC<ProfileTabProps> = ({ user }) => {
...
-  }, [blocker, checkConfirmation, t])
+    handleBlocker().catch((error: string) => {
+      dispatch(
+        openAlert({
+          severity: snackbarVariants.error,
+          message: error
+        })
+      )
+    })
+  }, [blocker, checkConfirmation, t, dispatch])
```
Comment (mhevyk) [inline src/containers/edit-profile/profile-tab/ProfileTab.tsx:80]: Please double check if all dependencies here are needed. For example `t`

File: src/containers/edit-profile/profile-tab/profile-tab-form/ProfileTabForm.tsx [hunk 2]
```diff
@@ -64,18 +64,15 @@ const ProfileTabForm: FC<ProfileTabFormProps> = ({
...
-          src: resizedPhoto,
-          name: photo.name
-        })
-      })
-      .catch(console.error)
+    const resizedPhoto = await imageResize(originalPhotoPath, photoSizes)
+    handleNonInputValueChange('photo', {
+      src: resizedPhoto,
+      name: photo.name
+    })
```
Comment (mhevyk) [inline src/containers/edit-profile/profile-tab/profile-tab-form/ProfileTabForm.tsx:75]: What will happen if `imageResize` fails? Maybe use `try/catch` here?

**Comment by mhevyk** [comment]
Please rebase your branch with new changes to avoid sonar and linter errors

File: src/containers/edit-profile/profile-tab/ProfileTab.tsx (hunk 1 above)
Comment (OliaHrytsak) [reply src/containers/edit-profile/profile-tab/ProfileTab.tsx:80]: This useEffect is build in such way that all this dependencies are needed
image

File: src/containers/edit-profile/profile-tab/profile-tab-form/ProfileTabForm.tsx (hunk 2 above)
Comment (OliaHrytsak) [reply src/containers/edit-profile/profile-tab/profile-tab-form/ProfileTabForm.tsx:75]: I've adder try/catch block an simulate error. Now looks like that
image

**Review by mhevyk — APPROVED** [review]

**Review by GeorgeShvab — APPROVED** [review]

**Review by dmtrth25 — APPROVED** [review]

**Review by byavdoshniak — APPROVED** [review]

**Pull request merged**